tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -1523,6 +1533,9 @@ bool PickBestAlgorithm(CudnnConvParams& params,
     params.algo_id = best_result.algorithm().algo_id();
     params.tensor_ops_enabled = best_result.algorithm().tensor_ops_enabled();
     params.best_result_bytes_used = best_result_bytes_used;
+#ifndef TENSORFLOW_USE_ROCM
+    params.workspace_size = best_result_bytes_used;
```

Comment:
nit: why is this different for CUDA and ROCM (at L1496)?

Reply:
I just do not have a workable ROCM device and have no way to verify the correctness. Thus I leave the ROCM part code as it is.